### PR TITLE
[13.0][MIG] stock_secondary_unit

### PR DESCRIPTION
**Pull request by ps-tubtim**
This PR is waiting for https://github.com/OCA/product-attribute/pull/537, you can always use this for readability.
- [x] [13.0][MIG] product secondary unit

### CODE REVIEW

File: stock_secondary_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Secondary Unit",
+    "summary": "Get product quantities in a secondary unit",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
```
Comment (sergio-teruel) [inline stock_secondary_unit/__manifest__.py]: Change status to "Production/Stable" and add maintainers key

File: stock_secondary_unit/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        string="Secondary Qty", digits="Product Unit of Measure"
+    )
+
+
+class StockMove(models.Model):
+    _inherit = ["stock.move", "stock.secondary.unit.mixin"]
+    _name = "stock.move"
+
+    def _merge_moves_fields(self):
+        res = super(StockMove, self)._merge_moves_fields()
```
Comment (sergio-teruel) [inline stock_secondary_unit/models/stock_move.py]: Change to super()._mer....

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: stock_secondary_unit/models/product.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    _name = "stock.product.secondary.unit"
+    _description = "Stock Product Secondary Unit"
+
+    secondary_unit_qty_available = fields.Float(
+        string="Quantity On Hand (2Unit)",
+        compute="_compute_secondary_unit_qty_available",
+        digits="Product Unit of Measure",
+    )
+
+    def _compute_secondary_unit_qty_available(self):
```
Comment (sergio-teruel) [inline stock_secondary_unit/models/product.py:17]:
```suggestion
    def _compute_secondary_unit_qty_available(self):
            for product in self:
                if not product.stock_secondary_uom_id:
                    product.secondary_unit_qty_available = 0.0
                else:
                    qty = product.qty_available / (product.stock_secondary_uom_id.factor or 1.0)
                    product.secondary_unit_qty_available = float_round(qty, precision_rounding=product.uom_id.rounding)

    
```

Comment (sergio-teruel) [reply stock_secondary_unit/models/product.py:17]: in V13 for computed non stored fields you always must be assign a value

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: stock_secondary_unit/__manifest__.py (hunk 1 above)
Comment (ps-tubtim) [reply stock_secondary_unit/__manifest__.py]: Edited.

File: stock_secondary_unit/models/stock_move.py (hunk 2 above)
Comment (ps-tubtim) [reply stock_secondary_unit/models/stock_move.py]: Edited.

**Comment by ps-tubtim** [comment]
@sergio-teruel Thank you for suggestion.

**Review by sergio-teruel — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Thank!! @ps-tubtim

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**